### PR TITLE
Layout on assembled components

### PR DESCRIPTION
**Pull request by MichaelDimitras**
@zacharyhill @Andyderson @TomGeshury 

I refactored everyone's components to use the webpack css loader I was using for my component.  There are going to be a lot of lines changed because of this, but they're all syntactical.

I refactored my react code to render a Layout component at the top level with everyone's components rendered inside of that along with screenshots of components we didn't do for this project.

### CODE REVIEW

File: client/src/Layout.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <div className={style.description}>
+          <Description id={props.params.id} />
+          <div className={style.boughtTogether} >
+          </div>
+        </div>
+        <div className={style.buyPhoto}>
+        </div>
+      </div>
+        <Related id={props.params.id} />
+        <div className={style.quesitons} >
```
Comment (Andyderson) [inline client/src/Layout.jsx]: Spelling error for questions*

File: client/src/Routes.jsx [hunk 2]
```diff
@@ -1,28 +1,14 @@
 import React from 'react';
```
Comment (Andyderson) [inline client/src/Routes.jsx:1]: What's the reasoning behind separating the React Routers and the components? Was it strictly separation of concern?

**Review by Andyderson — APPROVED** [review]
Everything looks clean with the refactoring of webpack css loader! Couple comments, nothing that's a major problem. 

Possible client change with the refactoring of the misspelled style.question.

File: client/src/Layout.jsx (hunk 1 above)
Comment (MichaelDimitras) [reply client/src/Layout.jsx]: Good catch

File: client/src/Routes.jsx (hunk 2 above)
Comment (MichaelDimitras) [reply client/src/Routes.jsx:1]: Yeah I had the main component in the file originally, but moved it into Layout.jsx as it became more detailed

**Pull request merged**